test/test_diag.cpp

```diff
@@ -593,6 +595,7 @@ TEST_F (DiagTest, DiagRunTest)
       TEST_SLOT_IDENT,
       TEST_SUBSLOT_IDENT,
       TEST_DIAG_USI_CUSTOM,
+      7,
       (uint8_t *)"Bjarn2");
```

Comment:
Replace "Bjarn" with something anonymous. Search for and replace "Bjarn"

Reply:
Done